homework/tai_nguyen/141_linked_list_cycle.ts

```diff
@@ -0,0 +1,21 @@
+function detectCycle(head: ListNode | null): ListNode | null {
```

Comment:
is this sol for https://leetcode.com/problems/linked-list-cycle-ii/ ?

Reply:
Oh, thanks @quanghoang33. I miss it and commit code of another similar problem.
Let me update this file :)